contracts/OtokenFactory.sol

```diff
@@ -0,0 +1,177 @@
+pragma solidity =0.6.10;
+
+import {Spawner} from "./packages/Spawner.sol";
+import {IAddressBook} from "./interfaces/IAddressBook.sol";
+import {IWhitelistModule} from "./interfaces/IWhitelistModule.sol";
```

Comment:
can we rename these to WhitlelistModuleInterface and AddressBookInterface ?

Reply:
I'm gonna rename to WhitelistInterface. (The term Module doesn't really add anything and it will become too long. )